PostHog/PostHogRemoteConfig.swift

```diff
@@ -76,10 +76,16 @@ class PostHogRemoteConfig {
             dispatchQueue.async {
                 self.reloadRemoteConfig { [weak self] remoteConfig in
                     guard let self else { return }
+
                     let hasFeatureFlags = remoteConfig?[self.hasFeatureFlagsKey] as? Bool == true
-                    let preloadFeatureFlags = self.config.preloadFeatureFlags
-                    // reload feature flags if not previously loaded
-                    if hasFeatureFlags, preloadFeatureFlags {
+
+                    // if server responds with `hasFeatureFlags: false`, then there are no more active flags on the account
+                    guard hasFeatureFlags else {
+                        return clearFeatureFlags()
```

Comment:
i think we should also set `loadingFeatureFlags: true` here since its kinda loaded, we know there's nothing server side

Reply:
oh yeah my bad, i meant the same as on Android `isFeatureFlagsLoaded` but I've mistaken the var name, we don't use it here.
so IMO we should notify though because its the same as loaded, we know there's nothing, and we're intentionally not calling the decide API, which means the app might be 'awaiting' for the callback/notification and it'll never arrive.